draft-ietf-quic-multipath.md

```diff
@@ -620,6 +620,23 @@ to abandon a path for any reason, for example, removing a hole from
 the sequence of Path IDs in use. This is not an error. The endpoint that
 receive such a PATH_ABANDON frame must treat it as specified in {{path-close}}.
 
+### Closing Duplicate Paths
```

Comment:
I think I would rather call this section something like "Using multiple paths on the same 4-tuple" and move it into the "Implementation Considerations" section?

Reply:
I agree that this might be a better place. We are going to end up with fuzzy language, so implementation considerations is more adequate.

Reply:
Just moved the text to the implementation section.